packages/expo-notifications/android/src/main/java/expo/modules/notifications/service/interfaces/FirebaseMessagingDelegate.kt

```diff
@@ -0,0 +1,17 @@
+package expo.modules.notifications.service.interfaces
+
+import com.google.firebase.messaging.RemoteMessage
+import expo.modules.notifications.service.NotificationsService
+
+interface FirebaseMessagingDelegate {
```

Comment:
Do you mind adding some comment for this interface? 🤔 
A bund of words that would address my initial confusion: `what's this interface delegate about?`